File: src/main/java/kr/mjc/rachel/basicsyntax/ForLoop.java
package kr.mjc.rachel.basicsyntax;

import java.util.List;

public class ForLoop {
  public static void main(String[] args) {
    List<String> items = List.of("apple", "banana", "kiwifruit");
    for (String item : items) {
      System.out.println(item);
    }
  }
}

File: src/main/kotlin/kr/mjc/jacob/basicsyntax/ForLoop.kt
/*
 * https://kotlinlang.org/docs/basic-syntax.html#for-loop
 */
package kr.mjc.jacob.basicsyntax

fun main() {
  val items = listOf("apple", "banana", "kiwifruit")
  for (item in items) {
    println(item)
  }
}